cfme/utils/appliance/implementations/ui.py

```diff
@@ -201,8 +201,9 @@ def check_for_badness(self, fn, _tries, nav_args, *args, **kwargs):
             # self.log_message('Op is a Nop! ({})'.format(fn.__name__))
             return
 
-        if self.VIEW:
-            self.view.flush_widget_cache()
+        # TODO: Uncomment after resolving the issue in widgetastic. Shouldn't be needed though :)
```

Comment:
might want to include link to WT PR if this actually gets merged: 
https://github.com/RedHatQE/widgetastic.core/pull/67